src/Race.js

```diff
@@ -0,0 +1,65 @@
+import Car from './Car.js';
+
+import { parseCars } from './utils/parser.js';
+import { sortScore } from './utils/sort.js';
+import { validateCars } from './utils/validator/car.js';
+import { validateCount } from './utils/validator/count.js';
+import { THRESHOLD } from './constants/threshold.js';
+import { Random } from '@woowacourse/mission-utils';
+import { getCars, getCount } from './utils/view/input.js';
+import {
+	showEmpty,
+	showRaceStatus,
+	showRaceWinner,
+	showResultMessage,
+} from './utils/view/output.js';
+
+export default class Race {
+	#cars;
+	#count;
+	#winner;
+
+	async init() {
+		const cars = await getCars();
+		const count = await getCount();
+
+		const carsArray = parseCars(cars);
+
+		validateCars(carsArray);
+		validateCount(count);
+
+		this.#cars = carsArray.map((name) => new Car(name));
+		this.#count = count;
+	}
+
+	start() {
+		showEmpty();
+		showResultMessage();
+		this.#round();
+		this.#selectWinner();
+		showRaceWinner(this.#winner);
+	}
+
+	#round() {
+		for (let i = 0; i < this.#count; i++) {
+			this.#race();
+			showRaceStatus(this.#cars);
+		}
```

Comment:
for 말고 고차함수로도 반복문을 돌리는 방법이 있는데 아래와 같은 방법도 있어요!
```suggestion
		Array.from({ length: this.#count}).forEach(() => {
			this.#race();
			showRaceStatus(this.#cars);
		})
```

Reply:
메서드명이 단순히 round면 현재 라운드를 보여주는건지, 라운드가 시작하려는건지, 라운드로 뭘 하려는건지 파악하기가 어려운데 `progressNextRound` 와 같이 조금 더 의미가 명확한 메서드명을 사용하면 좋을거 같아요

Reply:
상수값이라 어떻게 고차함수를 써야할지 몰랐는데 `Array.from({ length: this.#count})`과 같은 방법이 있었군요.. 야무진거 하나 배워 갑니다!! 👍